helm/frost-server/values.yaml

```diff
@@ -193,14 +197,14 @@ frost:
     # FROST-Server MQTT deployment settings
     enabled: true
     annotations: {}
-    serviceType: NodePort
+    serviceType: ClusterIP
```

Comment:
I agree with this change, but since this is a breaking change we must be careful with the release.